Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM ubuntu:latest
+FROM --platform=linux/amd64 ubuntu:latest
 MAINTAINER Chris Collins <collins.christopher@gmail.com>
 
-ENV PKGS git x11vnc wget python python-numpy unzip xvfb curl tar bzip2 runit
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=America/New_York
+
+ENV PKGS git x11vnc wget python3 python-is-python3 unzip xvfb curl tar bzip2 runit
```

Comment:
changed `python` to `python3 python-is-python3`, did not add back `python-numpy`, which I saw added a warning about performance, but otherwise didn't appear to affect